### PR TITLE
Refactor doc

### PR DESCRIPTION
**Pull request by benbovy**

### CODE REVIEW

**Comment by willirath** [comment]
I've modified the rtd webhook to not only trigger on pushes, but also on PRs (wasn't enabled until a few minutes ago). With #25, I get a status report from rtd building the pr. (And as I re-used the commits from this branch, I guess there's a re-computation of all checks here as well?)

File: doc/api.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Dataset.xoak
+------------
+
+This accessor extends :py:class:`xarray.Dataset` with all the methods and
+properties listed below. Proper use of this accessor should be like:
+
+.. code-block:: python
+
+   >>> import xarray as xr         # first import xarray
+   >>> import xsimlab              # import xoak (the 'xoak' accessor is registered)
```
Comment (willirath) [inline doc/api.rst:19]:
```suggestion
   >>> import xoak                 # import xoak (the 'xoak' accessor is registered)
```

Comment (benbovy) [reply doc/api.rst:19]: Ah yes copy/paste remnant.

**Comment by benbovy** [comment]
I refactored and moved the notebooks in `doc/examples` so that they can be both run with binder and shown in the documentation.

@willirath if CI passes (including doc build) this is ready to merge for me, if you want to have another look...

**Comment by willirath** [comment]
Looks great! Merging.

**Pull request merged**